src/Symfony/Component/PropertyInfo/Extractor/PhpDocExtractor.php

```diff
@@ -53,6 +53,10 @@ class PhpDocExtractor implements PropertyDescriptionExtractorInterface, Property
      */
     public function __construct(DocBlockFactoryInterface $docBlockFactory = null, array $mutatorPrefixes = null, array $accessorPrefixes = null, array $arrayMutatorPrefixes = null)
     {
+        if (!class_exists(DocBlockFactoryInterface::class)) {
```

Comment:
Shouldn't this be `interface_exists`?

http://php.net/manual/en/function.interface-exists.php

Reply:
good catch, in fact I wanted to use the `DocBlockFactory` class here as that's what is used below